src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -1177,13 +1177,30 @@ public boolean isVisibilityDropdownSeparatorHiddenForNewQuestion() {
         return isVisibilityDropdownSeparatorHidden(NEW_QUESTION_NUM);
     }
 
+    private String getFirstVisibilityMessage(int questionNumber) {
```

Comment:
Looks like this method is only used once in `verifyVisibilityMessageIsDisplayed`. Just move the contents to that method body?